app/views/steps/completion/confirmation/show.en.html.erb

```diff
@@ -1,14 +1,22 @@
 <% title 'Application submitted' %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <div class="grid_content_header govuk-box-highlight">
-      <h1 class="app__heading heading-xlarge gv-u-heading-xxlarge">Your application has been submitted</h1>
-      <p class="display-block util_mt-none util_mb-none">Your reference code is:</p>
-      <h3 class="heading-medium util_mt-none util_mb-medium"><%= current_c100_application.reference_code %></h3>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
+        Your application
+        <br>
+        has been submitted
+      </h1>
+      <div class="govuk-panel__body">
+        Your reference code is:
+        <br>
```

Comment:
However here, so the text does not wrap too much, we can remove the `<br>` but use:
<span class="app-util--no-wrap">Your reference code is:</span>